docs/cli/dotnet-publish.md

```diff
@@ -0,0 +1,66 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--output]  
+    [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` will compile the application, read through its dependencies specified in `project.json` and publish the resulting set of files to a directory. 
+This directory will contain the assemblies, the runtime as well as the executable version of the application. This directory can then be moved to a different machine and the application will be able to run regardless of existence of any other runtime. 
+
+The native dependencies of the runtime are not packaged with the application. This means that the target machine needs to have the native dependencies installed in order for the application to run.  
+
```

Comment:
What, exactly, does this mean? If I have the Kestrel package in my graph, it won't package the ones for the platform I am publishing for? Does that mean Kestrel has to somehow exist on the target machine in a well known cache? Does this work for things outside of the shared library? This bit is very confusing.